doc/docs/master/enterprise/auth/authorization/index.md

```diff
@@ -42,54 +42,42 @@ Pachyderm has a number of predefined roles granting permissions to its Resources
 Those Roles are listed here in no specific hierarchical order. 
 Some might inherit a set of permissions from another.
 
-- **repoReader**: Can consume data from a repo, but cannot edit them.
+### Repo Roles
+
+These roles can be granted at the repo level or at the cluster level - if a user is granted a role at the cluster level, it applies to all repos, including any new repos that are created after the grant. 
+
+- **repoReader**: A repoReader can consume data from a repo, but cannot edit them.
 repoReader can execute commands such as `pachctl get file` and
 `pachctl list file`, as well as create pipelines that use data
 from this repo. 
 
-- **repoWriter**: Can read and modify data in a repository by
+- **repoWriter**: A repoWriter can read and modify data in a repo by
 adding, deleting, or updating the files in the repo. The
 `repoWriter` role can perform operations such as `pachctl put file` or
 `pachctl delete commit`...
 
-- **repoOwner**: Additionaly to having the repoReader and the repoWriter Roles,
-a RepoOwner can grant permission to users on the Repository.
-
-    !!! Note
-        repoReader, repoWriter, and repoOwner can be set at all levels: cluster, and repo. 
-
-- **secretAdmin**: A secretAdmin has the ability to create, update, delete Kubernetes secrets on a cluster.
-
-    !!! Note
-        secretAdmin is set at the cluster level. 
+- **repoOwner**: A repoOwner can read and modify daat in a repo, 
+update the role bindings for that repo, and delete the repo.
 
+### Cluster Roles
 
-- **clusterAdmin**: A clusterAdmin can perform any
-action on the cluster including appointing other clusterAdmins.
-By default, the activation of Auth (`pachctl auth activate`) creates a Root User
-with irrevocable ClusterAdmin rights. This Role must be set at the cluster level only.
+These roles are only applicable at the cluster level. `clusterAdmin` is a catch-all role which allows a user to perform any operation on the cluster, while the others allow delegation of specific privileges depending on a users needs.
 
-    !!! Info
-        When Pachyderm Auth is not enabled,
-        all users are clusterAdmins.
```

Comment:
When auth isn't enabled we let every user do everything (we don't do any auth checks), but we also disable certain APIs. I think saying all users have clusterAdmin is kind of confusing for users.